apps/web/package.json

```diff
@@ -120,6 +120,7 @@
 		"react-popper-tooltip": "^4.4.2",
 		"react-resizable-panels": "^2.0.19",
 		"recoil": "^0.7.7",
```

Comment:
@NdekoCode if you switched from recoil to jotai, why you did not removed "recoild" package? Or NOT all is fixed yet?